.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  push:
+    branches: [main]
+    paths: [package.json]
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    outputs:
+      release_created:  ${{ steps.release.outputs.release_created }}
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
Are we OK with using a `v3` instead of a hash?

Reply:
Given in this case it's a Google-owned repository the risk is probably acceptable, compared to an arbitrary GH user

Reply:
Hum I'm not sure trusting Google more than individuals is very fair, I think it makes more sense to draw the line at GitHub (because they're the ones who manage the runner anyway) and treat any other non-Node.js with more caution. No strong feelings either way though.